Integrations/integration-BitDam.yml

```diff
@@ -111,11 +117,14 @@ script:
 
 
     def verdict_to_dbotscore(verdict):
-        if VERDICT_CLEAN == verdict:
+        if VERDICT_APPROVED == verdict:
             return DBOTSCORE_CLEAN
         elif VERDICT_MALICIOUS == verdict:
             return DBOTSCORE_MALICIOUS
-        return None
+        elif VERDICT_SCANNING == verdict:
+            return DBOTSCORE_UNKNOWN
+        else:
+            return DBOTSCORE_UNKNOWN
```

Comment:
If VERDICT_SCANNING == verdict the return value is DBOTSCORE_UNKNOWN, as well as in the "else" case.
Might as well delete the "VERDICT_SCANNING == verdict " case.